### ISSUE TITLE
Refactor: tests/resolvers/MembershipRequest

### ISSUE THREAD

**Issue by anshgoyalevil** [PalisadoesFoundation/talawa-api#970]
This is a child issue to the issue listed here: #965 

- Code need to be refactored for file ```tests/resolvers/MembershipRequest```
- We will need the API to be refactored for all methods, classes and/or functions found in this file for testing to be correctly executed.
- When complete, all all methods, classes and/or functions in the refactored file will need to be tested.
- Refer to the parent issue on getting an insight on how this issue need to be resolved.

## IMPORTANT:
Please refer to the parent issue on how to refactor the code correctly.

[Parent Issue](https://github.com/PalisadoesFoundation/talawa-api/issues/965)

## PR Acceptance Criteria
- When complete this file must show 100% coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found by visting this page](https://app.codecov.io/gh/PalisadoesFoundation/talawa-api?displayType=list). Login using your GitHub credentials.
- Create your own codecov.io to help with testing.
- The PR you create will show a report for the code coverage for the file you have added. You can also use that as a guide.

**Comment by anshgoyalevil** [comment]
Assign this to me

**Comment by aashimawadhwa** [comment]
I am willing to work on this issue can you please assign this to me.

**Comment by noman2002** [comment]
@anshgoyalevil 2 issues assigned to you. Please get the PRs merged before asking for more.

**Comment by nishant3721** [comment]
@noman2002 I would like to work on it!

**Comment by anshgoyalevil** [comment]
> @noman2002 I would like to work on it!

This issue is already assigned to @aashimawadhwa

**Comment by anshgoyalevil** [comment]
@aashimawadhwa Are you working on this?

**Comment by noman2002** [comment]
@anshgoyalevil You have various issues assigned to you. Please follow the procedure and get the PRs merged. I won't be able to assign any new issue until all the PRs are merged. I have informed you so many times about this.

**Comment by aashimawadhwa** [comment]
@anshgoyalevil  yes I am working on this I have my exams going on right now, will make pr in a day or two.

**Comment by anshgoyalevil** [comment]
> @anshgoyalevil yes I am working on this, will make pr in a day or two.

Okay. Good luck.
